src/openrct2/management/Finance.h

```diff
@@ -10,14 +10,11 @@
 #pragma once
 
 #include "../common.h"
-#include "../peep/Staff.h"
 #include "Research.h"
 
-using rct_expenditure_type = int32_t;
-
-enum
+enum rct_expenditure_type : int32_t
```

Comment:
can we enum class this?

Reply:
That will have about 600+ edits then, I can still do it tho.

Reply:
I think it makes sense to do it for this because its discrete values that dont (normally) do any maths on the underlying type. If it looks to be a huge load of changes then just leave it for now.

Reply:
Its not too bad I think